### ISSUE TITLE
Observability knowledge entrypoint/index page.

### ISSUE THREAD

**Issue by bwplotka** [cncf/sig-observability#19]
As per today's SIG CNCF meeting, it would be awesome if we could give an entry point to the observability projects we represent.

Overall the goal would be to improve the user world, especially new joiners. See https://github.com/cncf/sig-observability/issues/18 for example questions new users have when entering the observability world, this time from metrics angle.

Acceptance Criteria:
* Friendly starting point, how to start learning about observability for cloud-native apps
* Instead of repeating, link to project's docs, (some nice search as well?)
* Establish a feedback channel in case of missing docs/answers with observability projects.

**Comment by ArthurSens** [comment]
Should that entrypoint focus on developers or anything that could be related to getting started with observability? e.g. , some organization that just wants to monitor its servers on a cloud provider and don't know how to start

I guess that both developers and end-users could get started by taking a look at existing tools at the [Observability landscape](https://landscape.cncf.io/category=observability-and-analysis&format=card-mode&grouping=category) and, if possible, some sort of observability [tech radar](https://www.cncf.io/blog/2020/06/12/introducing-the-cncf-technology-radar/).

Developers should want to dig deeper, like how does everything work internally, how their code impact scalability and user experience, but maybe that is too much for someone that wants to solve a small problem and don't need that much information

**Comment by bwplotka** [comment]
> Should that entrypoint focus on developers or anything that could be related to getting started with observability? e.g. , some organization that just wants to monitor its servers on a cloud provider and don't know how to start

Yes, good question. Ideally it will be starting point with FAQ-like style (: 

Proposed something we can iterate on here: https://github.com/cncf/sig-observability/pull/21 Some FAQ like approach looks nice, but feedback welcome.
